pkg/systemd/init.js

```diff
@@ -297,11 +297,15 @@ $(function() {
             });
```

Comment:
This "Show message on empty result search" commit should be squashed into the previous one.

Reply:
Sure. But will squash into the following one, as it makes more sense that way.